pkg/kubelet/eviction/helpers.go

```diff
@@ -162,3 +183,360 @@ func parseGracePeriods(expr string) (map[Signal]time.Duration, error) {
 	}
 	return results, nil
 }
+
+// podUsage aggregates usage of compute resources.
+// it supports the following memory and disk.
+func podUsage(podStats statsapi.PodStats) (api.ResourceList, error) {
+	disk := resource.Quantity{Format: resource.BinarySI}
+	memory := resource.Quantity{Format: resource.BinarySI}
+	for _, container := range podStats.Containers {
+		// disk usage (if known)
```

Comment:
I will leave a // TODO for volume tracking, we can handle that in a follow-on when we finish up disk.